session-2/config.py

```diff
@@ -6,4 +6,5 @@
     PUBLIC_TAG: "10.24.0.0/23",
     PRIVATE_TAG: "10.24.2.0/23"
 }
-DESTINATION = "0.0.0.0/0"
\ No newline at end of file
+DESTINATION = "0.0.0.0/0"
+CIDR="10.24.0.0/16"
```

Comment:
Are you using the same config for the VPC creation? otherwise, this is not required